src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Abs.txt

```diff
@@ -9,9 +9,6 @@
 >> Abs(2)
 2
 
->> Abs(1E+308)
```

Comment:
Why remove these tests?

Reply:
It was moved with others to FloatLarge.txt.  They can only be run when NumberIsFloat is in effect.  The rest of the Abs.txt file can be run both with and without NIF.